docs/source/reference/index.rst

```diff
@@ -35,4 +36,4 @@ HUD
 
     /reference/respawn/hud_menus
     /reference/respawn/hud_element_notation
-    /reference/respawn/hud_script
\ No newline at end of file
+    /reference/respawn/hud_script
```

Comment:
Am I going insane, what whitespace change is happening here? ^^

Reply:
github web fuckery